jenkins/codecov_response_metrics.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: UTF-8 -*-
+
+# Occasionally, pull requests never get updated with codecov results, which
+# can create a lot of difficulty in getting the pr merged. However, this is
+# purely anecdotal.
+
+# This script attempts to determine the extent of the issue. It scans a series
+# of repositories that are currently reporting coverage metrics to codecov,
+# and sees if any 'recently' touched pull requests have taken longer than a
+# specified time to receive their codecov github status (or have not received
+# it at all).
+
+# This script should be run periodically in order to get a good understanding
+# of the state of codecov response times.
+
+import os
+import sys
+import datetime
+import logging
+
+from github import Github
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+REPOS = [
+    'edx/edx-app-ios',
+    'edx/edx-enterprise',
+    'edx/ecommerce',
+    'edx/studio-frontend',
+    'edx/xqueue'
+    'edx/edx-gomatic',
+    'edx/edx-drf-extensions',
+    'edx/edx-analytics-dashboard',
+    'edx/edx-proctoring',
+    'edx/credentials',
+    'edx/course-discovery',
+    'edx/edx-video-pipeline',
+    'edx/edx-analytics-pipeline'
+    'edx/completion',
+    'edx/XBlock',
+    'edx/edx-enterprise-data',
+    'edx/edx-notes-api',
+    'edx/edx-app-android',
+    'edx/edx-video-worker',
```

Comment:
Why isn't edx-platform in this list?

Reply:
edx-platform is the only repo that doesn't run codecov/patch, so i was avoiding it while testing